ui/app/templates/rules.html

```diff
@@ -37,57 +43,116 @@
+        </span>
+      </div>
+      <i title="Product" ng-show="!rule.product && !rule.scheduled_change.product">no product</i>
+      <b title="Product">
+        <span ng-show="rule_id || !show_sc || !fieldIsChanging(rule, 'product')">{{ rule.product }}</span>
+        <s style="color: red" ng-show="!rule_id && show_sc && fieldIsChanging(rule, 'product')">{{ rule.product }}</s>
+        <span style="color: green" ng-show="!rule_id && show_sc && fieldIsChanging(rule, 'product')">{{ rule.scheduled_change.product }}</span>
+      </b>
       :
-      <i title="Channel" ng-show="!rule.channel">no channel</i>
-      <b title="Channel" ng-show="rule.channel" ng-bind-html="highlightSearch(rule.channel, 'channel')"></b>
+      <i title="Channel" ng-show="!rule.channel && !rule.scheduled_change.channel">no channel</i>
+      <b title="Channel">
+        <span ng-show="rule_id || !show_sc || !fieldIsChanging(rule, 'channel')">{{ rule.channel }}</span>
+        <s style="color: red" ng-show="!rule_id && show_sc && fieldIsChanging(rule, 'channel')">{{ rule.channel }}</s>
+        <span style="color: green" ng-show="!rule_id && show_sc && fieldIsChanging(rule, 'channel')">{{ rule.scheduled_change.channel }}</span>
+      </b>
     </h3>
   </div>
   <div class="panel-body">
+    <div ng-show="show_sc && rule.scheduled_change !== null && ! isEmpty(rule.scheduled_change.required_signoffs)">
+      <span style="float: left">
+        <h4>Before this change can be enacted, it must be signed off on by:</h4>
+        <ul>
+          <li ng-repeat="(rs_role, signoffs_required) in rule.scheduled_change.required_signoffs">
+            {{ signoffs_required }} member(s) of {{ rs_role }}
+          </li>
+        </ul>
+      </span>
+      <span style="float: right" ng-show="rule.scheduled_change !== null && ! isEmpty(rule.scheduled_change.signoffs)">
+        <h4>This change has already been signed off by:</h4>
+        <ul>
+          <li ng-repeat="(username, signoff_role) in rule.scheduled_change.signoffs">
+            {{ username }} ({{ signoff_role }})
+          </li>
+        </ul>
+      </span>
+    </div>
+    <hr style="clear: both" ng-show="show_sc && rule.scheduled_change !== null && ! isEmpty(rule.scheduled_change.required_signoffs)">
+
     <div class="left" style="float: left">
 
-      <h5 title="Mapping">
-        Mapping: <span ng-show="rule.mapping">
-            <b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b>
+      <h5 title="Mapping" ng-show="rule.mapping || (show_sc && rule.scheduled_change !== null && rule.scheduled_change.mapping)">
+        Mapping:
+          <span ng-show="rule.mapping">
+            <b ng-style="!rule_id && show_sc && rule.scheduled_change !== null && rule.mapping !== rule.scheduled_change.mapping ? {'text-decoration': 'line-through', 'color': 'red'}: ''">
```

Comment:
I tried to schedule a deletion, and I found the UI clearer, even though mapping is struck out. I guess that's because every field is too.

This looks great!